assets/csidriveroperators/openstack-cinder/hypershift/mgmt/generated/apps_v1_deployment_openstack-cinder-csi-driver-operator.yaml

```diff
@@ -0,0 +1,122 @@
+              matchExpressions:
+              - key: hypershift.openshift.io/cluster
+                operator: In
+                values:
+                - ${CONTROLPLANE_NAMESPACE}
+            weight: 100
+        podAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - podAffinityTerm:
+              labelSelector:
+                matchLabels:
+                  hypershift.openshift.io/hosted-control-plane: ${CONTROLPLANE_NAMESPACE}
+              topologyKey: kubernetes.io/hostname
+            weight: 100
+      containers:
+      - args:
+        - start
+        - -v=${LOG_LEVEL}
+        - --guest-kubeconfig=/etc/guest-kubeconfig/kubeconfig
+        env:
+        - name: DRIVER_CONTROL_PLANE_IMAGE
+          value: ${DRIVER_CONTROL_PLANE_IMAGE}
+        - name: LIVENESS_PROBE_CONTROL_PLANE_IMAGE
+          value: ${LIVENESS_PROBE_CONTROL_PLANE_IMAGE}
+        - name: DRIVER_IMAGE
+          value: ${DRIVER_IMAGE}
+        - name: PROVISIONER_IMAGE
+          value: ${PROVISIONER_IMAGE}
+        - name: ATTACHER_IMAGE
+          value: ${ATTACHER_IMAGE}
+        - name: RESIZER_IMAGE
+          value: ${RESIZER_IMAGE}
+        - name: SNAPSHOTTER_IMAGE
+          value: ${SNAPSHOTTER_IMAGE}
+        - name: NODE_DRIVER_REGISTRAR_IMAGE
+          value: ${NODE_DRIVER_REGISTRAR_IMAGE}
+        - name: LIVENESS_PROBE_IMAGE
+          value: ${LIVENESS_PROBE_IMAGE}
+        - name: KUBE_RBAC_PROXY_IMAGE
+          value: ${KUBE_RBAC_PROXY_IMAGE}
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        image: ${OPERATOR_IMAGE}
+        imagePullPolicy: IfNotPresent
+        name: openstack-cinder-csi-driver-operator
+        resources:
+          requests:
+            cpu: 10m
+            memory: 50Mi
+        terminationMessagePolicy: FallbackToLogsOnError
+        volumeMounts:
+        - mountPath: /etc/guest-kubeconfig
+          name: guest-kubeconfig
+        - mountPath: /etc/openstack
+          name: secret-cinderplugin
+          readOnly: true
+        - mountPath: /etc/kubernetes/static-pod-resources/configmaps/cloud-config
+          name: cacert
+      priorityClassName: hypershift-control-plane
+      serviceAccountName: openstack-cinder-csi-driver-operator
+      tolerations:
+      - key: CriticalAddonsOnly
+        operator: Exists
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
+        operator: Exists
+      - key: hypershift.openshift.io/control-plane
+        operator: Exists
+      - key: hypershift.openshift.io/cluster
+        operator: Equal
+        value: ${CONTROLPLANE_NAMESPACE}
+      volumes:
+      - name: guest-kubeconfig
+        secret:
+          secretName: service-network-admin-kubeconfig
+      - name: secret-cinderplugin
+        secret:
+          items:
+          - key: clouds.yaml
+            path: clouds.yaml
+          secretName: openstack-cloud-credentials
```

Comment:
A `CredentialsRequest` needs to be added to the hosted cluster, otherwise no `openstack-cloud-credentials` will exist

Reply:
Actually, this seems to be addressed by this other PR https://github.com/openshift/hypershift/pull/4936/files#diff-0de67b02a8275f7a8227b3af5101786e70e5c651b852a87f6160f6563a9071d6R281-R284